integration/spark-common/src/main/scala/org/apache/carbondata/spark/CarbonOption.scala

```diff
@@ -31,6 +31,10 @@ class CarbonOption(options: Map[String, String]) {
 
   def tablePath: String = s"$dbName/$tableName"
 
+  def allowExisting: String = options.getOrElse("allowExisting", "true")
```

Comment:
a basic option when creating table. Create table IF NOT EXISTS... user can make choice on this  IF NOT EXISTS part. if there is no need to add this, i will remove it, pls give ur suggestion.